frontend_lib/i18next.scanner/fr/translation.json

```
@@ -83,5 +90,7 @@
   "User + create shared spaces, add members in shared spaces": "Utilisateur + créer des espaces partagés, ajouter des membres dans les espaces partagés",
   "Shared space manager": "Responsable",
   "Refresh?": "Rafraîchir ?",
-  "If you refresh, you will lose the current changes": "Si vous vous rafraîchissez, vous perdrez les modifications actuelles"
+  "If you refresh, you will lose the current changes": "Si vous vous rafraîchissez, vous perdrez les modifications actuelles",
```

```suggestion
  "If you refresh, you will lose the current changes": "Si vous rafraîchissez, vous perdrez les modifications actuelles",
```

Suggestion: If you refresh the page